pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -62,27 +61,24 @@ stdenv.mkDerivation rec {
       --subst-var-by libc_bin ${lib.getBin stdenv.cc.libc} \
       --subst-var-by libc_dev ${lib.getDev stdenv.cc.libc} \
       --subst-var-by libc_lib ${lib.getLib stdenv.cc.libc}
-  ''
```

Comment:
If you didn't do this one change you could get around the mass rebuild btw

Reply:
Thats fine for me :)

Reply:
🙈